### PR TITLE
Task5 (Edit Mode)

### PR DESCRIPTION
**Pull request by MonkeyKingBar-bit**
image
image
Edit:
image
Save:
image
Delete:
image

### CODE REVIEW

**Review by Fancy322vav — COMMENTED** [review]
This looks GREAT. I enjoy reviewing your PRs. 
But functionality is not quite correct. You cards 'jump' while editing and saving (I edit one card and after saving it 'jumps' to another one). Also please add a global 'Cancel' btn to cancel all the changes in the edit mode and close edit mode.
Screenshot 2021-09-07 at 18 32 40
Screenshot 2021-09-07 at 18 32 47

File: src/components/Card/Button/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface ButtonProps {
+   name: string;
+   disabled: any;
+   onClick: any;
```
Comment (Fancy322vav) [inline src/components/Card/Button/Button.tsx:6]: Please try not to use 'any' type.

**Comment by Fancy322vav** [comment]
If you're struggled with fixing this functionality, fell free to ping me and I will help you.

File: src/components/Card/Button/Button.tsx (hunk 1 above)
Comment (Fancy322vav) [reply src/components/Card/Button/Button.tsx:6]: It shouldn't be at your code.

Comment (MonkeyKingBar-bit) [reply src/components/Card/Button/Button.tsx:6]: I'm understand it, but if i write another type, that get an error from eslint.

**Pull request merged**